frontend/src/App.tsx

```diff
@@ -576,8 +578,29 @@ export class App extends PureComponent<Props, State> {
     this.handleSessionStateChanged(initialize.sessionState)
   }
 
+  createThemeHash = (themeInput: CustomThemeConfig): string | null => {
+    if (!themeInput) {
+      return null
+    }
+
+    const themeInputEntries = Object.entries(themeInput)
+    themeInputEntries.sort()
```

Comment:
TIL sorting arrays of arrays shockingly works...might be due to toString, which I guess could present challenges, but I guess what matters more is a consistent order than a specific order.

Reply:
The behavior is actually sane (as long as the arrays are all the same length) where it sorts by the first entry, using the second to tiebreak, and so on. Let me add a comment here, though since this is far from obvious